src/D2L.CodeStyle.Analyzers/ApiUsage/DependencyInjection/DependencyRegistrationsAnalyzer.cs

```diff
@@ -208,13 +208,22 @@ ITypeSymbol type
 			return ImmutableArray.Create( registration.DependencyType );
 		}
 
-		private bool TryGetDependenciesFromConstructor( ITypeSymbol type, out ImmutableArray<ITypeSymbol> dependencies  ) {
-			var ctor = type.GetMembers()
+		private bool TryGetInjectableConstructor( ITypeSymbol type, out IMethodSymbol injectableConstructor ) {

```

Comment:
Dunno if there's a reason everything can't be `INamedTypeSymbol` (don't expect so), but it has a `.Constructors` if we did